rules_jvm_export/jvm_export/jvm_assembly.bzl

```diff
@@ -66,6 +66,17 @@ def runtime_output_jar(target):
     if JavaInfo in target:
         if len(target[JavaInfo].runtime_output_jars) == 1:
             return target[JavaInfo].runtime_output_jars[0]
+        elif len(target[JavaInfo].runtime_output_jars) == 2:
+            if target[JavaInfo].runtime_output_jars[0].path.endswith("_java.jar"):
+                return target[JavaInfo].runtime_output_jars[0]
+            elif target[JavaInfo].runtime_output_jars[1].path.endswith("_java.jar"):
+                return target[JavaInfo].runtime_output_jars[1]
+            else:
+                fail(
+                    "expected size 1, or the file name ends with _java.jar, but runtime_output_jars in {} was {}".format(
+                        target, target[JavaInfo].runtime_output_jars
+                    )
+                )
```

Comment:
could this be simplified to
```
for runtime_output_jar in target[JavaInfo].runtime_output_jars:
  if runtime_output_jar.path.endswith("_java.jar"):
    return jar

fail(...)

Reply:
Done - 29bf7a9d69a381970e9ac988db999c736e92fefb